lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -195,6 +207,46 @@ public function render_passwordless_enabled( $args = array() ) {
 		);
 	}
 
+	/**
+	 * Render form field and description for the `auto_login` option.
+	 * IMPORTANT: Internal callback use only, do not call this function directly!
+	 *
+	 * @param array $args - callback args passed in from add_settings_field().
+	 *
+	 * @see WP_Auth0_Admin_Generic::init_option_section()
+	 * @see add_settings_field()
+	 */
+	public function render_auto_login( $args = array() ) {
+		$this->render_switch( $args['label_for'], $args['opt_name'], 'wpa0_auto_login_method' );
+		$this->render_field_description(
+			__( 'Use the Universal Login Page (ULP) for authentication. ', 'wp-auth0' ) .
```

Comment:
this is the only place you use `(ULP)` (in this PR at least). not a deal breaker for me, your call.

Reply:
@luisrudge - Used on line 225 below as well.

Reply:
damn tunnel vision. I was looking in other code blocks 😬